res/schema.xml

```diff
@@ -582,4 +582,19 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Suggestion tables added.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS suggestion_library (
+        id INTEGER,
+        artist varchar(48),
+        title varchar(48),
+        playcount INTEGER,
+        match FLOAT,
```

Comment:
I am considering if it makes sense to perfix the last.fm specific columns with Last.FM This way we may reuse the entry for other providers after adding their measures.

Reply:
That makes sense, as you said we can have different providers.